internal/assets/contents/versioninfo.json

```diff
@@ -0,0 +1,41 @@
+{
+  "FixedFileInfo": {
+    "FileVersion": {
+      "Major": 0,
+      "Minor": 0,
+      "Patch": 0
+    },
+    "ProductVersion": {
+      "Major": 0,
+      "Minor": 0,
+      "Patch": 0
+    },
+    "FileFlagsMask": "3f",
+    "FileFlags ": "00",
+    "FileOS": "040004",
+    "FileType": "01",
+    "FileSubType": "00"
+  },
+  "StringFileInfo": {
+    "Comments": "",
+    "CompanyName": "ActiveState Software Inc.",
+    "FileDescription": "",
+    "FileVersion": "",
+    "InternalName": "",
+    "LegalCopyright": "© ActiveState Software, Inc. 2024",
```

Comment:
Are we going to have to remember to change this every year? If so, please change the generator to modify this field. Perhaps omit the year here and append it there.